src/main/java/Bundle.properties

```diff
@@ -1701,6 +1701,8 @@ file.addreplace.error.dataset_is_null=The dataset cannot be null.
 file.addreplace.error.dataset_id_is_null=The dataset ID cannot be null.
 find.dataset.error.dataset_id_is_null=When accessing a dataset based on Persistent ID, a {0} query parameter must be present.
 find.dataset.error.dataset.not.found.persistentId=Dataset with Persistent ID {0} not found.
+find.dataset.error.dataset.not.found.id=Data Project with ID {0} not found.
+find.dataset.error.dataset.not.found.bad.id=Bad data project ID number: {0}.
```

Comment:
We call them "datasets", not "data projects". I don't think we should merge this as-is.